programs/gateway/src/state/registrar.rs

```diff
@@ -25,6 +25,10 @@ pub struct Registrar {
     /// The Gatekeeper Network represents the "Pass Type" that a 
     /// user must present.
     pub gatekeeper_network: Pubkey,
+    
+    /// If the plugin is one in a sequence, this is the previous plugin
+    /// If set, then update_voter_weight_record will expect a voter_weight_record owned by this registrar
+    pub predecessor_plugin_registrar: Option<Pubkey>,
```

Comment:
We would also need an instruction to configure both `previous_voter_weight_plugin` and `gatekeeper_network` for existing `Registrar`